website/docs/cli/import/importability.mdx

```diff
@@ -9,7 +9,9 @@ description: |-
 Each resource in OpenTofu must implement some basic logic to become
 importable. As a result, you cannot import all OpenTofu resources.
 
-The resources that you can import are documented at the bottom of
-each resource documentation page in the [public Terraform Registry](https://registry.terraform.io/). If you have issues importing a resource, report an issue in the relevant provider repository.
+The resources that you can import for a provider are listed in that
```

Comment:
I'm pondering if we can get rid of the passive voice here. Something along the lines of:

> Please see the the provider's specific documentation for a list of resources you can import.